ArduCopter/mode_turtle.cpp

```diff
@@ -142,6 +142,11 @@ void ModeTurtle::run()
 // actually write values to the motors
 void ModeTurtle::output_to_motors()
 {
+    if (!motors->armed() || !motors->get_interlock()) {
```

Comment:
I'm not sure about this part of the change though because this means we are not updating any RC outputs on this loop which we always have done before.

Reply:
Yes, your right, I have re-worked.